nicolegooden/indecision-maker
# Display result page om

#### What's this PR do?  

This PR displays all relevant information about the activities filtered by the script. It uses all user's answers to show the suggested activities in the `ResultPage` component and allows the user to select a specific activity and learn more about it by `clicking` on the `choose me` button.

#### Where should the reviewer start? 

components:
 - `<ResultPage>`
- `<DetialsPage>`

#### How should this be manually tested?  

- Make sure the server is running on `localhost:3000`
- Run `npm start` to run the application on `localhost:5000`
- Click on the `find activity button`
-  Complete the form
- Click on the `submit` button
- Click on the `skip` or `choose me` button

#### Any background context you want to provide?  
We initially planned to create individual components that would render all the activities separately, however as the project evolved as a team. we decided to create a single component called `<ResultPage>` that would display the all suggested activities.  The next step was to create a second component called `<DetailsPage>` that would render the information about the user's chosen activity.

#### What are the relevant tickets?  
- closes #3 
- closes #4 
- closes #5 
- closese #10 
- closes #18 
#### Screenshots (if appropriate) 
![Screen Shot 2020-11-04 at 1 53 01 PM](https://user-images.githubusercontent.com/56229864/98166699-15da5780-1ea5-11eb-8f90-60eb813b04ed.png)
![Screen Shot 2020-11-04 at 1 53 30 PM](https://user-images.githubusercontent.com/56229864/98166735-238fdd00-1ea5-11eb-9df8-4e5ac00cbafa.png)
#### Questions:
